WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController+SectionHelpers.swift

```diff
@@ -56,7 +56,21 @@ extension BlogDetailsViewController {
     }
 
     @objc func shouldShowStats() -> Bool {
-        return JetpackFeaturesRemovalCoordinator.shouldShowJetpackFeatures()
+        return JetpackFeaturesRemovalCoordinator.shouldShowJetpackFeatures() && JetpackFeaturesRemovalCoordinator.jetpackFeaturesEnabled()
```

Comment:
[Nit] This should be enough:

```suggestion
        return JetpackFeaturesRemovalCoordinator.jetpackFeaturesEnabled()
```

If Jetpack features are enabled, `shouldShowJetpackFeatures` will always be true. (The opposite is not true, though)

Reply:
Good point!